Hooks/Chat.cs

```diff
@@ -21,53 +18,51 @@ public static class Chat
     /// </summary>
     public static event ChatEventHandler? OnChatMessage;
```

Comment:
Might be good to add a remarks to explain the compromise for this hook, or maybe even a deprecation notice to this functionality. I understood why this hook existed given the native detour, but it's not as obvious to me outside of compatibility.

Reply:
Not sure what you're looking for here. I have a comment in the PR. The chat feature isn't deprecated, it's just changed internally.

Reply:
Never mind if only internal change. I read the implementation as intentionally not supporting the canceling in the way it use to work, resolving in favor of PR discussion of testing. The original thinking was that if mods depended on the message not going out in the original channel then they could be broken and it wouldn't just be an internal change.